script/PresignPauseFromJson.s.sol

```diff
@@ -55,6 +55,7 @@ contract PresignPauseFromJson is MultisigBuilder, JsonTxBuilderBase {
         override
     {
         checkStateDiff(accesses);
+        require(false, "_postCheck not implemented");
```

Comment:
the code below is dead. any use keeping it around?

Reply:
Hm good point, @maurelian since this is `PresignPauseFromJson`, I imagine the post checks will be the same every time so maybe we can just delete the `require` here?

Reply:
Oh, good callout. I agree this `require` statement can be deleted.